sdk/ml/azure-ai-ml/azure/ai/ml/_ml_client.py

```diff
@@ -12,56 +12,32 @@
 from pathlib import Path
 from typing import Any, Optional, Tuple, TypeVar, Union
 
-from azure.core.credentials import TokenCredential
-from azure.core.polling import LROPoller
-
 from azure.ai.ml._azure_environments import (
     CloudArgumentKeys,
+    _add_cloud_to_environments,
     _get_base_url_from_metadata,
     _get_cloud_information_from_metadata,
     _get_default_cloud_name,
     _set_cloud,
-    _add_cloud_to_environments,
 )
 from azure.ai.ml._file_utils.file_utils import traverse_up_path_and_find_file
 from azure.ai.ml._restclient.v2020_09_01_dataplanepreview import (
     AzureMachineLearningWorkspaces as ServiceClient092020DataplanePreview,
 )
-from azure.ai.ml._restclient.v2022_01_01_preview import (
-    AzureMachineLearningWorkspaces as ServiceClient012022Preview,
```

Comment:
Why changed the format?

Reply:
it's updated by pre-commit before I updated the pre-commit excludes. Will revert